testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.5
+_pkgver=v2.1.5
```

Comment:
No need for two version variables. Just add the v to the source url:
```
$pkgname-$pkgver.tar.gz::https://binaries.cockroachdb.com/$pkgname-v$pkgver.src.tgz
```

Reply:
Ah, just noticed that - was left over from pre-release testing, I'll nuke that too